src/ledger/NetworkConfig.cpp

```diff
@@ -1061,6 +1062,18 @@ SorobanNetworkConfig::isValidConfigSettingEntry(ConfigSettingEntry const& cfg,
                     cfg.contractCompute().txMaxInstructions &&
                 cfg.contractCompute().txMemoryLimit >=
                     MinimumSorobanNetworkConfig::MEMORY_LIMIT;
+
+        // In protocol V23+, check that ledgerMaxInstructions * clusters won't
```

Comment:
I think this is going to break the 'unlimited resources' use case of core for standalone builds. We already check the tx sets for overflows of the actual per-cluster resources, as well as the total resource sums. In combination with the saturating multiplication for the tx queue, I think we should be fine with any configuration.